.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/recommended",
+    "plugin:react/jsx-runtime",
+    "plugin:react/recommended",
+    "plugin:storybook/recommended",
```

Comment:
Too much copy/paste .. pretty sure we don't have any storybook here (nor react, etc etc)